### PR TITLE
Popup - Add layer ID and feature ID as data attributes

### PR DESCRIPTION
**Pull request by Gustry**
According to what I understood, using data-attributes looks is better ? 
https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes

I see in many places, that in the code, this hidden input value is used.
And then this value need to be split with `.` to separate layer ID and feature ID :

```html
<input class="lizmap-popup-layer-feature-id" value="polygons_1e227060_6105_4a4a_92f2_11863ebe65f1.0" type="hidden">
```

I'm proposing
image

```javascript
lizMap.events.on({
    lizmappopupdisplayed: function (popup) {
        // It should be #popupcontent
        let popupContainer = document.getElementById(popup.containerId);

        if (!popupContainer) return false;

        Array.from(popupContainer.querySelectorAll('div.lizmapPopupContent .lizmapPopupSingleFeature')).map(element => {
            console.log(element.dataset.featureId);
            console.log(element.dataset.layerId);
        });
    }
});
```

Linked to #4990

Need to check for raster

### CODE REVIEW

**Comment by mdouchin** [comment]
I fully agree with this modification. We will have break in Javascript scripts trying to find the previous hidden field, but the needed modification will not be hard to do

**Comment by Gustry** [comment]
For a raster, it gives : 

```xml
<div data-layer-id="local_raster_layer_c4c2ec5e_7567_476b_bf78_2b7c64f32615" data-feature-id="" class="lizmapPopupSingleFeature">
```

Do we want it empty, or not available ?

File: lizmap/modules/view/templates/popup.tpl [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="lizmapPopupSingleFeature">
+<div class="lizmapPopupSingleFeature" data-feature-id="{$featureId}" data-layer-id="{$layerId}">
```
[the remark this answers is not in the document]
Comment (Gustry) [reply lizmap/modules/view/templates/popup.tpl:1]: It was already done in my local copy ;-)

But I was asking if we want it empty or not. So I understand YES ;-)

**Comment by Gustry** [comment]
PR ready for review @mind84 @rldhont @mdouchin @nboisteault

**Review by nboisteault — APPROVED** [review]

**Comment by mind84** [comment]
I'll probably have to modify several custom `js` but it's a more than welcome change :smile: 

Thanks!

**Comment by Gustry** [comment]
No problem.
Indeed, I'm also wondering how we will advertise these deprecated changes.
(and when ...)

**Pull request merged**